Added Field level Security + Changed App Tab Order

jonathanthomas37 opened the pull request:
Created SSN field for Order and Policies with custom validation rules and edited app to show order and policies.

jonathanthomas37 committed:
Added Field level Security

Created SSN field for Order and Policies with custom validation rules and edited app to show order and policies

rdylang reviewed (changes requested):
Deploy errors:
```
Error  Order-Order Layout            In field: field - no CustomField named Order.Social_Security_Number__c found (34:26)       
Error  Admin                         In field: field - no CustomField named Order.Social_Security_Number__c found
Error  Custom%3A Agent               In field: field - no CustomField named Order.Social_Security_Number__c found
Error  Custom%3A Field Trainer       In field: field - no CustomField named Order.Social_Security_Number__c found
Error  Custom%3A Unit Sales Manager  In field: field - no CustomField named Order.Social_Security_Number__c found
```

jonathanthomas37 committed:
Fixed Errors

added CustomField meta tag to manifest to specifically add Order.Social_Security_Number__c field to order objects folder
